File: VRP/VRPCore/src/inputTools/inputFile/Point.cpp
#include "Point.h"

Point::Point(int x = 0, int y = 0) :
	x(x), y(y)
{
}

Point::Point(const Point& p)
{
	this->x = p.getX();
	this->y = p.getY();
}

int Point::getX() const
{
	return x;
}

void Point::setX(int x)
{
	this->x = x;
}

int Point::getY() const
{
	return y;
}

void Point::setY(int y)
{
	this->y = y;
}

File: JEE/src/main/java/vrp/jee/domain/Point.java
package vrp.jee.domain;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlRootElement;

/**
 *
 * @author Maciej Radzikowski <maciej@radzikowski.com.pl>
 */
@XmlRootElement
@XmlAccessorType(XmlAccessType.FIELD)
public class Point {

	private Integer x;

	private Integer y;

	public Point() {
	}

	public Point(Integer x, Integer y) {
		this.x = x;
		this.y = y;
	}

	public Integer getX() {
		return x;
	}

	public void setX(Integer x) {
		this.x = x;
	}

	public Integer getY() {
		return y;
	}

	public void setY(Integer y) {
		this.y = y;
	}
    
}
